docs/configuring-archival.md

```diff
@@ -0,0 +1,135 @@
+---
+id: configuring-archival
+title: Configuring Archival
+---
+ 
+## What is Archival?
+
+Archival is a feature that automatically moves Workflow Event Histories from Temporal's normal persistence storage to a long-term blob store after the Workflow retention period expires.
+
+This feature enables Event Histories to be stored indefinitely while not overwhelming the persistence store.
+
+The Archival feature can be customized per Namespace, but must be enabled and configured at the Cluster level as well.
+
+:::note
+
+**Archival only supports Event Histories.**
+
+You may see some infrastructure in place to support archiving visibility records. However, the functionality is not yet supported and visibility records are currently deleted after the Workflow retention period.
+
+:::
+
+## Why use Archival?
+
+There are two main reasons you may want to keep Event Histories after the retention period has past:
+
+1. **Compliance:** For legal reasons Event Histories may need to be stored for a long period of time.
+2. **Debugging:** Old Event Histories may help with debugging.
+
+:::info
+
+**Want to turn off the Archival feature?**
+
+Set `archival.history.state` and `namespaceDefaults.archival.history.state` to `disabled`.
+
+:::
+
+## How to use Archival
+
+There are four main components to the Archival feature that you need to be aware of.
+
+1. **Archival feature configuration**: The Archival feature is controlled by the `archival` configuration in the `config/development.yaml` file.
+2. **Archive Provider**: The location where the Event Histories are archived to is called the Archive Provider. Common providers are S3, GCloud, Kafka, and the local file system.
```

Comment:
when documenting things that have key-value pairs

**hello:** words

It's not necessary to start with boilerplate like "the"

**Archive Provider**: location where Event Histories will be archived. Supported providers are S3, GCloud, Kafka, and the local file system.